### DISCUSSION TITLE
[Bug]: 'dev' docker images no longer start from commit 7031c47f (02/06/2024 15:47)

### DISCUSSION THREAD

**Discussion by ubawurinna** [blakeblackshear/frigate#11969] [Report a Bug] [answered]
### Describe the problem you are having

Docker image startup log complains about **tempio** command not found  
_Docker image startup log_
```
[INFO] Starting NGINX...
./run.user: line 84: tempio: command not found
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
[INFO] Service NGINX exited with code 127 (by signal 0)
```

The **tempio** command is called from a nginx script
_docker/main/rootfs/etc/s6-overlay/s6-rc.d/nginx/run_
```
# build templates for optional TLS support
python3 /usr/local/nginx/get_tls_settings.py | \
    tempio  -template /usr/local/nginx/templates/listen.gotmpl \
            -out /usr/local/nginx/conf/listen.conf
```

The problem is due to the fact that the **tempio** command folder is not in the **PATH**.
_Bash inside docker_
```
echo ${PATH}
/usr/lib/btbn-ffmpeg/bin:/usr/local/go2rtc/bin:/usr/local/nginx/sbin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
But the **tempio** command folder is defined in the **PATH** in the Dockerfile
_docker/main/Dockerfile_
```
ENV PATH="/usr/lib/btbn-ffmpeg/bin:/usr/local/go2rtc/bin:/usr/local/tempio/bin:/usr/local/nginx/sbin:${PATH}"
```
I don't know why the **tempio** command folder is no longer defined in **PATH** once the docker image has been created.  
The Docker images come from the URL ghcr.io/blakeblackshear/frigate:dev-xxxxxxx  
If the **tempio** command is called with the full path in the nginx script, the docker image starts.

### Steps to reproduce

run docker image dev-7031c47 or later

### Version

0.14.0-0d862d6

### Frigate config file

```yaml
Useless
```

### Relevant log output

```shell
[INFO] Starting NGINX...
./run.user: line 84: tempio: command not found
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
[INFO] Service NGINX exited with code 127 (by signal 0)
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wireless

### Camera make and model

Tapo C510W

**Comment by hawkeye217** [comment, marked as answer]
This is probably a duplicate of https://github.com/blakeblackshear/frigate/discussions/11755

Are you using Portainer?

**Reply by ubawurinna** [reply]
yes it's totally a duplicate.
Same problem, same solution.
Destroy the portainer frigate stack and recreate from scratch
